fashion-messaging-app-ui/src/App.jsx

```diff
@@ -68,6 +64,7 @@ useEffect(() => {
       const fashionPhotos = fetchPromises.filter(photo => photo !== undefined);
       setPhotos(fashionPhotos);
       setNoResults(fashionPhotos.length === 0);
+      setLoading(false);
```

Comment:
I changed this because I noticed the loading state was only applicable in the first API and I wanted it to have a loading state when users search for images